### PR TITLE
columns name modified

### PR DESCRIPTION
**Pull request by matheusnatanb**

### CODE REVIEW

**Review by ytcsc — CHANGES_REQUESTED** [review]
The code changes are good and I'm inclined to approve the pull request. However, before we proceed with merging, please address the feedback provided in each individual file's comments. Once you've made these adjustments, the code should be ready for merging. Thank you for your efforts!

File: Software/Experiments/StepResponse.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-time [ms],sp1_abs [°C],sp2_abs [°C],sp1_rel [°C],sp2_rel [°C],h1_mul_noise,h2_mul_noise,fan_mul_noise,h1_add_noise,h2_add_noise,fan_add_noise,h1_neg_sat,h2_neg_sat,fan_neg_sat,h1_pos_sat,h2_pos_sat,fan_pos_sat
+TIME,SP1_ABS,SP2_ABS,SP1_REL,SP2_REL,H1_MUL_NOISE,H2_MUL_NOISE,F_MUL_NOISE,H1_ADD_NOISE,H2_ADD_NOISE,F_ADD_NOISE,H1_NEG_SAT,H2_NEG_SAT,F_NEG_SAT,H1_POS_SAT,H2_POS_SAT,F_POS_SAT
 0,,,,,1,1,1,0,0,0,0,0,0,100,100,100
 10000,,,,,1,1,1,10,10,0,0,0,0,100,100,100
-1810000,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+20000,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
```
[the remark this answers is not in the document]
Comment (matheusnatanb) [reply Software/Experiments/StepResponse.csv]: Done

File: Software/enums.py
[the remark this answers is not in the document]
Comment (matheusnatanb) [reply Software/enums.py:1]: Done

File: Software/Modules/TCL_Experiment.py [hunk 2]
```diff
@@ -24,7 +24,8 @@
 import numpy as np
 import pathlib
 import time
-import datetime
+from datetime import datetime
+from enums import CSVColumns
```
[the remark this answers is not in the document]
Comment (matheusnatanb) [reply Software/Modules/TCL_Experiment.py]: you're right

**Review by ytcsc — APPROVED** [review]

**Pull request merged**